pkg/services/store/httpobjectstore/service.go

```diff
@@ -0,0 +1,369 @@
+package httpobjectstore
```

Comment:
nit, but what do you think about keeping it under `object`? maybe  `services/store/object/http/service.go`?

Reply:
done